build.ps1

```diff
@@ -32,7 +32,7 @@ $simulateReleaseBuild = $null
 if (-not([bool]::TryParse($env:IsReleaseBuild, [ref] $isReleaseBuild) -and
```

Comment:
Something to consider:
To set a release build, you could do the following: 1) Set a variable in the pipeline 2) Or analyze the branch from which the build is being executed. 

Is there a reason why we need both of these variable `$env:SimulateReleaseBuild` and `$env:IsReleaseBuild`?

Reply:
Yes, `$env:IsReleaseBuild` is evaluated at runtime to determine if the branch is a release branch, i.e. the branch name is of the form `release_2.0*`. `$env:SimulateReleaseBuild` is a variable to be overridden when the build will not be used for a release, but we want to check that the build process is working properly for release builds (specifically, to check release-specific build steps like SBOM generation or signing). What do you think? Should we combine these in some way?